keps/sig-node/3983-drop-in-configuration/README.md

```diff
@@ -277,7 +274,7 @@ No
 
 ###### How can an operator determine if the feature is in use by workloads?
 
-Workloads do not directly consume this feature, it is for cluster admins during kubelet configuration. Administrators can check the kubelet feature flag metric `kubernetes_feature_enabled` to see if this is enabled.
+Workloads do not directly consume this feature, it is for cluster admins during kubelet configuration.
 
 
 ###### How can someone using this feature know that it is working for their instance?
```

Comment:
Could you expand this section to provide a little more information on how users can determine this?